bundles/ch.elexis.core.data/src/ch/elexis/data/CsvUser.java

```diff
@@ -0,0 +1,202 @@
+package ch.elexis.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ch.elexis.core.data.service.CoreModelServiceHolder;
+import ch.elexis.core.model.IContact;
+import ch.elexis.core.model.IRole;
+import ch.elexis.core.model.IUser;
+import ch.elexis.core.model.IXid;
+import ch.elexis.core.model.RoleConstants;
+
+public class CsvUser implements IUser {
```

Comment:
Bitte in Test-Bundle auslagern. (Müssen wir noch klären wie das architektonisch gehen soll)

Reply:
mach ich.